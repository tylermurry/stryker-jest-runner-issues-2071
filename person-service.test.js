const { generatePerson } = require('./person-service');

describe('Person Service Tests', () => {

  it('should generate a normal person', () => {
    expect(generatePerson('John', 'Smith', 30)).toMatchSnapshot();
  });

  it('should generate a retired person', () => {
    expect(generatePerson('Jane', 'Smith', 75)).toMatchSnapshot();
  });

  it('should generate a minor', () => {
    expect(generatePerson('Joe', 'Smith', 16)).toMatchSnapshot();
  });

});
