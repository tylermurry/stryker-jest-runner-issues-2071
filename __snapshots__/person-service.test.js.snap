// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Person Service Tests should generate a minor 1`] = `
Object {
  "firstName": "Joe",
  "fullName": "Joe Smith",
  "lastName": "Smith",
  "minor": true,
  "retired": false,
}
`;

exports[`Person Service Tests should generate a normal person 1`] = `
Object {
  "firstName": "John",
  "fullName": "John Smith",
  "lastName": "Smith",
  "minor": false,
  "retired": false,
}
`;

exports[`Person Service Tests should generate a retired person 1`] = `
Object {
  "firstName": "Jane",
  "fullName": "Jane Smith",
  "lastName": "Smith",
  "minor": false,
  "retired": true,
}
`;
